src/modules/checks/suites/execution_node.py

```diff
@@ -48,13 +48,25 @@ def check_events_range_availability(web3, blockstamp, deposit_contract):
     list(get_events_in_range(deposit_contract.events.DepositEvent, blockstamp.block_number, latest_block.number))
 
 
-def check_events_week_range_availability(web3, deposit_contract):
-    """Check that execution-client able to get event logs a week ago"""
+@pytest.mark.parametrize('range_', [8 * 225 * 32, 32 * 225 * 32], ids=['week', 'month'])
+def check_events_week_range_availability(web3, deposit_contract, range_):
```

Comment:
Does it make sense to define a `DepositContract` in the provider module?

Reply:
it is used only for this file and only in testing purpose. not sure about changing the provider module for this